### PR TITLE
updated students section

### PR DESCRIPTION
**Pull request by HussainArif12**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`
- Preview Link
-

### CODE REVIEW

**Comment by HussainArif12** [comment]
How do i squash commits?

File: _config.yml [hunk 1]
```diff
@@ -1 +1,3 @@
 baseurl: ""
+
+theme: jekyll-theme-minimal
```
Comment (realslimshanky) [inline _config.yml]: Add a new line after this.

File: _data/students.yml [hunk 2]
```diff
@@ -29,3 +29,6 @@
   github: STEdirisuriya
   image: supun.png
 
+- name: Hussain Arif
+  github: HussainArif12
+  image: nick photo.png
```
Comment (realslimshanky) [inline _data/students.yml]: The name of the image should be more informative, make it about the person whose image it is.

**Comment by sarnava1** [comment]
@realslimshanky Ohh sure!! Did not know about it!! Thanks for informing.

**Comment by abishekvashok** [comment]
@realslimshanky we don't need a real photo of the student. Please don't ask that. @sarvana1 just rename your image and it's good to get a merge.

**Comment by HussainArif12** [comment]
So, I dont need to squash the commits right?

**Pull request closed without merging**